### PR TITLE
Added dbus watchdog timer support and additonal check for null SSE st…

### PR DESCRIPTION
**Pull request by srinivasadatari**
Add dbus watchdog support and better handling for null SSE stream reads.

### CODE REVIEW

File: cmd/idracsdkapps/splunkapp/main.go [hunk 1]
```diff
@@ -144,7 +146,8 @@ func logToSplunk(events []*SplunkEvent) {
 	if resp.Body != nil {
 		resp.Body.Close()
 	}
-	log.Printf("Sent to Splunk. Got back %d", resp.StatusCode)
```
Comment (windsparks33) [inline cmd/idracsdkapps/splunkapp/main.go]: We'll want to keep the log method.
Can you change fmt.Print* to log.Print*?

File: cmd/idracsdkapps/splunkapp/main.go [hunk 2]
```diff
@@ -74,7 +76,7 @@ type SplunkEvent struct {
 var configStringsMu sync.RWMutex
 var configStrings = map[string]string{
 	"splunkURL": "http://10.244.123.128:8088",
-	"splunkKey": "a270b34f-afbd-4d5a-a2c2-520cad1ce2a7",
```
Comment (windsparks33) [inline cmd/idracsdkapps/splunkapp/main.go:77]: splunk key will be different for each user, let's drop this change.

File: cmd/idracsdkapps/splunkapp/main.go [hunk 3]
```diff
@@ -263,13 +280,13 @@ func main() {
 
 	for {
 
-		req, err := http.NewRequest("GET", "https://10.239.37.136/redfish/v1/SSE?$filter=EventFormatType%20eq%20MetricReport", nil)
+		req, err := http.NewRequest("GET", "https://100.69.104.132/redfish/v1/SSE?$filter=EventFormatType%20eq%20MetricReport", nil)
```
Comment (windsparks33) [inline cmd/idracsdkapps/splunkapp/main.go:282]: If I remember correctly this code will run on the iDRAC.  The HTTP Request can be changed to target the iDRAC it is on with a switch from 100.69.104.132 to "localhost"

File: cmd/idracsdkapps/splunkapp/main.go (hunk 2 above)
Comment (srinivasadatari) [reply cmd/idracsdkapps/splunkapp/main.go:77]: I think we should hold on to this for now. The proper thing to do would be to make this part of the user config work, so these get configured when the plugin boots.

File: cmd/idracsdkapps/splunkapp/main.go [hunk 4]
```diff
@@ -263,13 +280,13 @@ func main() {
...
 		if err != nil {
 			fmt.Println("NewRequest GET", req, err)
 		}
 		fmt.Println("NewRequest GET", req, err)
 
-		req.SetBasicAuth("root", "VMware1!")
+		req.SetBasicAuth("root", "Dell1234")
```
Comment (windsparks33) [inline cmd/idracsdkapps/splunkapp/main.go:288]: We should get the authentication piece via dbus.  Jitendra has some great code on this.  I'll dig it up.

File: cmd/idracsdkapps/splunkapp/main.go (hunk 3 above)
Comment (srinivasadatari) [reply cmd/idracsdkapps/splunkapp/main.go:282]: Again same comment as above. Should be part of user config changes.

File: cmd/idracsdkapps/splunkapp/main.go (hunk 4 above)
Comment (srinivasadatari) [reply cmd/idracsdkapps/splunkapp/main.go:288]: Same as above.

File: cmd/idracsdkapps/splunkapp/main.go (hunk 1 above)
Comment (srinivasadatari) [reply cmd/idracsdkapps/splunkapp/main.go]: Good catch. Changed it and pushed the change.

**Pull request merged**